assets/data-labeling/environments/data-labeling-sam/context/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM mcr.microsoft.com/azureml/aifx/stable-ubuntu2004-cu117-py38-torch1131:biweekly.202305.3
+
+# Install pip dependencies
+COPY requirements.txt .
+RUN pip install -r requirements.txt --no-cache-dir
+
+# Inference requirements
+COPY --from=mcr.microsoft.com/azureml/o16n-base/python-assets:20230419.v1 /artifacts /var/
```

Comment:
do you need this or just copied from some env?

Reply:
just copied, will remove.